packages/zoe/src/contracts/priceAggregator.js

```diff
@@ -217,11 +221,11 @@ const start = async zcf => {
 
       const paKit = makeOnewayPriceAuthorityKit({
         createQuote: makeCreateQuote(),
-        mathIn,
-        mathOut,
         notifier,
         quoteIssuer: quoteKit.issuer,
         timer,
+        actualBrandIn: brandIn,
+        actualBrandOut: brandOut,
```

Comment:
Many of the methods have `brandIn` or `brandOut` as parameters so they needed a different name. Open to suggestion, but because we often do a check of the `brandIn` passed in as a parameter compared to the `brandIn` that the priceAggregator actually has, I called it `actualBrandIn`